test/Twig/Tests/EnvironmentTest.php

```diff
@@ -145,10 +145,18 @@ public function testExtensionsAreNotInitializedWhenRenderingACompiledTemplate()
         // force compilation
         $twig = new Twig_Environment($loader = new Twig_Loader_Array(array('index' => '{{ foo }}')), $options);
         $cache = $twig->getCacheFilename('index');
-        if (!is_dir(dirname($cache))) {
-            mkdir(dirname($cache), 0777, true);
+        $dir = dirname($cache);
+        $created = false;
+        if (!is_dir($dir)) {
+            if (false === $created = @mkdir($dir, 0777, true)) {
+                $this->markTestSkipped(sprintf('Failed to create test cache directory at "%s".', $dir));
```

Comment:
well, being allowed to create files and folders in the system temp directory is a meaningful expectation IMO. If your user is not allowed to do that, I would consider that your have a broken system setup.